server/util/ongoingRegistration.js

```diff
@@ -0,0 +1,32 @@
+const { parseSemesterIntoYearSemesterNumber, findMatchedPeriod, SEMESTER_NUMBER } = require('./semesterUtils')
+
+const computeFirstRegistrationDate = ladokSemester => {
+  const registrationDates = { [SEMESTER_NUMBER.SPRING]: '09-15', [SEMESTER_NUMBER.AUTUMN]: '03-15' }
+  const summerRegistrationDate = '02-15'
```

Comment:
Maybe add

```
SEMESTER_NUMBER.SUMMER = 3
```

in imported `SEMESTER_NUMBER` and refactor the following to

```suggestion
const registrationDates = {
  [SEMESTER_NUMBER.SPRING]: '09-15',
  [SEMESTER_NUMBER.AUTUMN]: '03-15',
  [SEMESTER_NUMBER.SUMMER]: '02-15',
}

```

Reply:
Had a chat with Benni about this, and we weren't sure if we should add 'SUMMER' to SEMESTER_NUMBER or not as it's only used in certain places. I wouldn't mind adding it there though!

Reply:
Ok! Leave it as you like 🙏